### DISCUSSION TITLE
How to disable row selectable on tree checkbox component

### DISCUSSION THREAD

**Discussion by sh977218** [primefaces/community#1052] [PrimeNG] [answered]
I'm using v16 of Angular PrimeNG. 
In many case, there are going to be some actionable elements (link/dropdown etc) on the row. However, he link/dropdowns click always trigger the checkbox.  Is there any way to select the row by only check the checkbox?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can listen to the click and prevent default on the row if the event target is not a checkbox.
